Develop/assets/css/style.css

```diff
@@ -29,6 +29,15 @@ body {
   background-color: var(--outer-space);
 }
 
+/* new */
+header {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  text-align: center;
+  margin: 2%;
+}
+
```

Comment:
Makes header a flex-container. Flex-flow could be column instead of row. Justify-content does nothing here. Align text centers children elements' text.